build.sbt

```diff
@@ -119,7 +120,9 @@ lazy val testing = crossProject(JSPlatform, JVMPlatform)
   .settings(
     name := "log4cats-testing",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-effect" % catsEffectV
+      "org.typelevel" %%% "cats-effect"     % catsEffectV,
+      "ch.qos.logback"  % "logback-classic" % logbackClassicV % Test,
+      "org.specs2"     %% "specs2-core"     % specs2V         % Test
```

Comment:
I'd prefer to keep the tests in MUnit rather than introduce a second framework, but we can clean that up in a subsequent PR.

Reply:
Will do that later

Reply:
Done